opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx

```diff
@@ -214,16 +215,7 @@ const LeftBar = () => {
   const isGrantedToImport = useGranted([KNOWLEDGE_KNASKIMPORT]);
   const isGrantedToProcessing = useGranted([KNOWLEDGE_KNUPDATE, SETTINGS_SETACCESSES, CSVMAPPERS]);
   const isGrantedToSharing = useGranted([TAXIIAPI]);
-  const isGrantedToSettings = useGranted([
-    SETTINGS_SETPARAMETERS,
-    SETTINGS_SETACCESSES,
-    SETTINGS_SETMARKINGS,
-    SETTINGS_SETCUSTOMIZATION,
-    SETTINGS_SETLABELS,
-    SETTINGS_SECURITYACTIVITY,
-    SETTINGS_FILEINDEXING,
-    SETTINGS_SUPPORT,
-  ]);
+  const isGrantedToSettings = useGranted([SETTINGS]);
```

Comment:
Nop, SETTINGS is no longer an "available" capa, you can not use useGranted on that